bin/fin

```diff
@@ -3279,56 +3285,57 @@ update ()
 			sudo mv "$FIN_PATH_UPDATED" "$FIN_PATH"
 			exit
 		else
-			echo-rewrite "Updating fin... $FIN_VERSION ${green}[OK]${NC}"
+			echo-rewrite "[STEP 1/$TOTAL_STEPS] Updating fin... $FIN_VERSION ${green}[OK]${NC}"
 		fi
 	fi
 
-	# [Update - 2] Update default configs
+	# [STEP 2] Update stacks
+	echo-green "[STEP 2/$TOTAL_STEPS] Updating stack files..."
 	update_config_files
 
-	# [Update - 3] Update start script and shortcut
-	# TODO: deprecated, remove in September 2017
-	if is_windows; then
-		echo-green "Updating start script and shortcut..."
-		update_start_script_shortcut
-	fi
-
-	# [Update - 4] Update third party tools
+	# [STEP 3] Update third party tools
+	echo-green "[STEP 3/$TOTAL_STEPS] Updating tools..."
 	update_tools
 
-	# [Update - 5] Update system images
+	# [STEP 4] Update system images
 
-	# Always update system images during upgrade
+	echo-green "[STEP 4/$TOTAL_STEPS] Updating system images..."
 	if [[ ${UPGRADE_IN_PROGRESS} == 1 ]]; then
-		echo-green "Updating system images..."
+		# [4.1] Always update system images during upgrade
+		# On Win and Mac update only during upgrade because it is done in docker_machine_start
 		update_system_images
 		# Run cleanup to get rid of outdated images
 		cleanup
-	fi
+	else
+		# [4.2] On Linux update during install, but use sudo as user will not be in docker group yet.
+		if is_linux; then
+			# On Linux a subnet needs to be created first
+			configure_resolver_linux
+			# Override docker to run it via sudo
+			docker () {
+				sudo "$(which docker)" "$@"
+			}
+			update_system_images
+		fi
 
-	# On Linux also do it during install, but use sudo as user will not be in docker group yet.
-	# On Win and Mac it is done in docker_machine_start
-	if [[ ${UPGRADE_IN_PROGRESS} != 1 ]] && is_linux; then
-		# On Linux a subnet needs to be created first
-		configure_resolver_linux
-		echo-green "Updating system images..."
-		# Override docker to run it via sudo
-		docker () {
-			sudo "$(which docker)" "$@"
-		}
-		update_system_images
+		# [4.3] With native mode update during install, but no cleanup
+		if is_docker_native; then
+			update_system_images
+		fi
 	fi
 
-	# With native mode also do it during install
-	if [[ ${UPGRADE_IN_PROGRESS} != 1 ]] && is_docker_native; then
-		echo-green "Updating system images..."
-		update_system_images
+	# [STEP 5] Update project images
+	if [[ ${UPGRADE_IN_PROGRESS} == 1 ]]; then
+		echo-green "[STEP 5/$TOTAL_STEPS] Updating project images"
```

Comment:
This should say "Updating Docksal images", since it updates all Docksal images, not just those of a specific project